brat/examples/vec_map.brat

```diff
@@ -0,0 +1,15 @@
+ext "to_float" to_float(i :: Int) -> Float
```

Comment:
maybe this file would be better called "mono_vec_map.brat" (or monomorphic...)

Reply:
Also "list" rather than "vec"